src/model/kaniBinaryRunner.ts

```diff
@@ -7,16 +7,12 @@ import * as vscode from 'vscode';
 import { KaniArguments, KaniConstants, KaniResponse } from '../constants';
 import { getRootDir } from '../utils';
 import { runCargoKaniCommand } from './cargokaniBinaryRunner';
-import { CommandArgs } from './kaniCommand';
+import { CommandArgs, getKaniPath, options } from './kaniCommand';
```

Comment:
Suggest renaming to `exec_options` or `kani_options` if you're going to import it into other files. `options` could mean anything, I'd prefer a more descriptive name for global symbols.